src/credentials/status/on-chain-revocation.ts

```diff
@@ -32,13 +33,19 @@ export class OnChainResolver implements CredentialStatusResolver {
    * @returns Promise<RevocationStatus>
    */
   async getRevocationOnChain(credentialStatus: CredentialStatus): Promise<RevocationStatus> {
-    const { contractAddress, chainId, revocationNonce } = this.parseOnChainId(credentialStatus.id);
+    const { contractAddress, chainId, revocationNonce, issuer } = this.parseOnChainId(
```

Comment:
I guess we need to consume issuer not from url (theoretically it can have a contract address of another public issuer). we need to pass it as parameter.

Reply:
fixed.